modules/apps/27-interchain-accounts/controller/keeper/migrations.go

```diff
@@ -31,7 +33,7 @@ func (m Migrator) AssertChannelCapabilityMigrations(ctx sdk.Context) error {
 
 		isAuthenticated := m.keeper.scopedKeeper.AuthenticateCapability(ctx, cap, name)
 		if !isAuthenticated {
-			return sdkerrors.Wrapf(capabilitytypes.ErrCapabilityNotOwned, "expected capability owner: %s", types.SubModuleName)
+			return sdkerrors.Wrapf(capabilitytypes.ErrCapabilityNotOwned, "expected capability owner: %s", commitmenttypes.SubModuleName)
```

Comment:
Not sure how this ended up as `commitmenttypes` I see the import previously would've been incorrect also. 
It should be `controllertypes.SubModuleName`.

Reply:
Ah right, I was wondering about that actually, I changed the import to be inline with other imports in the file. I can update this now.

Reply:
I think the dangers of `types` as the import alias.